inspirehep/modules/forms/static/js/forms/inspire-form.js

```diff
@@ -468,12 +468,14 @@ define(function(require, exports, module) {
 
       set_status(tpl_status_saving());
 
-      $.ajax(
+      var promise = $.ajax(
```

Comment:
That's how it was before and while trying some things I separated it.

It is exactly the same :)